docs/user_guide/write_api/push_job.md

```diff
@@ -65,6 +67,8 @@ The user may choose to specify the following configs:
 - `targeted.region.push.list`: This config takes effect only when targeted region push flag is enabled. 
   Optionally specify a list of target region(s) to push data into. See full details at 
   [TARGETED_REGION_PUSH_LIST](https://venicedb.org/javadoc/com/linkedin/venice/hadoop/VenicePushJob.html#TARGETED_REGION_PUSH_LIST).
+- `post.validation.consumption`: Whether to perform post validation consumption after targeted region push is finished. 
```

Comment:
Add bit more detail like disable this feature to achieve a true single colo push?